Content.Client/State/GameScreenBase.cs

```diff
@@ -209,11 +209,12 @@ private void OnKeyBindStateChanged(BoundKeyEventArgs args)
             var mousePosWorld = EyeManager.ScreenToMap(args.PointerLocation);
             var entityToClick = GetEntityUnderPosition(mousePosWorld);
 
-            if (!MapManager.TryFindGridAt(mousePosWorld, out var grid))
-                grid = MapManager.GetDefaultGrid(mousePosWorld.MapId);
+            IMapGrid grid;
+            if (!MapManager.TryFindGridAt(mousePosWorld, out grid))
+                grid = null;
```

Comment:
Jesus this is annoying to look at.

Can't you just move the getting of the `FullInputCmdMessage` arg out so we don't need this?
or at least invert the logic.

```cs
IMapGrid grid = null;
MapManager.TryFindGridAt(mousePosWorld, out grid);
```

Reply:
Changed.